napari/utils/colormaps/colormap_utils.py

```diff
@@ -601,16 +601,18 @@ def ensure_colormap(colormap: ValidColormapArg) -> Colormap:
         elif isinstance(colormap, VispyColormap):
             # if a vispy colormap instance is provided, make sure we don't already
             # know about it before adding a new unnamed colormap
-            name = None
+            _name = None
```

Comment:
In python the variables in functions started with an underscore convention for marking the variable as unused. You should use `name_` instead of `_name`